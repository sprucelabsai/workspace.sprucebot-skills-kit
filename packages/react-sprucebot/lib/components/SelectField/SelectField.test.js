'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

require('jest-styled-components');

var _SelectField = require('./SelectField');

var _SelectField2 = _interopRequireDefault(_SelectField);

var _FormField = require('../FormField/FormField');

var _FormField2 = _interopRequireDefault(_FormField);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

test('it renders a FormField', function () {
	var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(
		_SelectField2.default,
		{ input: { value: 'test', onChange: jest.fn() }, meta: {} },
		_react2.default.createElement('option', { value: 'testval' })
	));

	expect(wrapper).toMatchSnapshot();

	expect(wrapper.find(_FormField2.default)).toHaveLength(1);
});