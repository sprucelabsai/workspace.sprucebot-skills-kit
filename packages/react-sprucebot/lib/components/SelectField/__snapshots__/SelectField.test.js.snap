// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders a FormField 1`] = `
<Field
  input={
    Object {
      "onChange": [MockFunction],
      "value": "test",
    }
  }
  meta={Object {}}
>
  <SelectField__StyledSelect>
    <option
      value="testval"
    />
  </SelectField__StyledSelect>
</Field>
`;
